### ISSUE TITLE
Seeds to make results reproducible

### ISSUE THREAD

**Issue by amonroymz** [MultiX-Amsterdam/text-data-module#5]
There is some randomness in the code, leading to different results for every execution. For example, when we train the neural network in task 7, the loss for each epoch vary. Should we use seeds so the results are reproducible?

<img width="439" alt="image" src="https://github.com/MultiX-Amsterdam/text-data-module/assets/149786124/7a55f08e-4c92-4ad9-ad42-f3f2635e10d1">

**Comment by yenchiah** [comment]
Maybe do not worry about this for now. We can leave it here for next year.
